cmd/init.go

```diff
@@ -37,6 +37,8 @@ type initCommandConfig struct {
 	noTemplate bool
 }
 
+var prjName string
```

Comment:
This needs to go into the `RootCommandConfig` or `initCommandConfig`- no more global vars.

Reply:
Moved into the initCommandConfig